README.md

```diff
@@ -32,11 +32,14 @@ pip install git+https://github.com/descriptinc/descript-audio-codec
 
 ### Weights
 Weights are released as part of this repo under MIT license.
-They are automatically downloaded when you first run `encode` or `decode` command. They can be cached locally with
+We release weights for models that can natively support 24kHz and 44.1 kHz sampling rates.
```

Comment:
Super small nit: I would write it together like "44.1kHz", to be consistent with the "24kHz" earlier.

Reply:
done